notebooks/community/vertex_endpoints/nvidia-triton/nvidia-triton-custom-container-prediction.ipynb

```diff
@@ -0,0 +1,1667 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        if delete_endpoint:
</pre><p>Could you please add <code>or os.getenv("IS_TESTING")</code> on all delete checks to ensure the CI tests don't leave behind resources? Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='101'/>

Reply:
Added additional condition on delete checks.